applications/FluidDynamicsHydraulicsApplication/tests/test_hydraulic_fluid_auxiliary_utilities.py

```diff
@@ -73,8 +41,38 @@ def testCalculateWettedArea(self):
         for node in inlet_skin_model_part.Nodes:
             node.SetValue(Kratos.AUX_DISTANCE, node.Z - level_set_z)
         WettedArea= KratosFluidHydraulics.HydraulicFluidAuxiliaryUtilities.CalculateWettedArea(inlet_skin_model_part,Kratos.INLET,Kratos.AUX_DISTANCE, False)
-        theoretical_wetted_area= 2.0*(level_set_z)
+        theoretical_wetted_area= 0.5*(level_set_z)
         self.assertAlmostEqual(WettedArea, theoretical_wetted_area, 12)
+  
+    def testCornerDetection(self):
```

Comment:
```suggestion
    def testFixCornerVelocity(self):
```
It is really convenient to call the test as the feature you're testing.